### ISSUE TITLE
PC Configuration Survey

### ISSUE THREAD

**Issue by GreenComfyTea** [GreenComfyTea/MHW-Better-Matchmaking#2]
Currently, there is an issue that the mod doesn't work in some circumstances. The community assumed it's due to AMD GPU. To prove or disprove this, I need to collect more information.

**Whether the mod works for you or not, please, dedicate a couple of minutes so I could improve the mod for everyone!**

Download and install [testing build](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/releases/tag/v2.0-amd-testing-v2), launch the game and perform one `Filter Search` for sessions, close the game.

> [!NOTE]
> Type `winver` in the windows search and run it to get your OS version.

> [!NOTE]
> Type `dxdiag` in the windows search and run it. Press `Save All Information...` to generate dxdiag report file.

> [!NOTE]
> `SharpPluginLoader.log` is located in `\Monster Hunter World\nativePC\plugins\CSharp\Loader\` folder.

`Template:`

```
**Is the mod working:**
**If not then what happens:**

**OS:**
**CPU:**
**RAM:**
**GPU:**
**GPU Driver Version:**
**Game Version:**
**Stracker's Loader Version:**
**SharpPluginLoader Version:**

**Dxdiag Report File:**
**SharpPluginLoader Log File:**

**All Installed Mods (with versions):**

<details>
<summary>Show/Hide</summary>

...
</details>

```

**Comment by GreenComfyTea** [comment]
**Is the mod working:** Yes

**OS:** Microsoft Windows 10 Enterprise 22H2 (OS Build 19045.4170)
**CPU:** Intel i9-9900k 8C/16T 3.6 GHz (OC 4.5 GHz)
**RAM:** 32 GB 3200 MHz
**GPU:** Nvidia RTX 3090 Ti Founders Edition (OC Core +135 MHz Memory +1100 MHz)
**GPU Driver Version:** v551.86
**Game Version:** v15.21.00
**Stracker's Loader Version:** v3.0.1
**SharpPluginLoader Version:** v0.0.4.1

**Dxdiag Report File:** [DxDiag.txt](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14721018/DxDiag.txt)
**SharpPluginLoader Log File:** [SharpPluginLoader.log](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14721120/SharpPluginLoader.log)

**All Installed Mods (with versions):**
<details>
  <summary>Show/Hide</summary>

- [RTSS Overlay v7.3.4](https://www.guru3d.com/page/rivatuner-rtss-homepage/)
- [Reshade 6.0.0](https://reshade.me/) with SMAA.fx
- [Stracker's Loader v3.0.1](https://www.nexusmods.com/monsterhunterworld/mods/1982)
- [Performance Booster and Plugin Extender v1.3](https://www.nexusmods.com/monsterhunterworld/mods/3473)
- [HunterPie v2.10.0.134](https://www.nexusmods.com/monsterhunterrise/mods/181)
- [HelloWorld (an overlay that shows lots and lots of stuff) v10.4](https://www.nexusmods.com/monsterhunterworld/mods/142)
- [Bow Spread Pattern Fix v1.0](https://www.nexusmods.com/monsterhunterworld/mods/6914)
- [Tic Rate Fix v1.0](https://www.nexusmods.com/monsterhunterworld/mods/3474)
- [Guiding Lands Gathering Indicator v1.12.11.01](https://www.nexusmods.com/monsterhunterworld/mods/1986)
- [Gems Sorted by Name (Alphabetized Gems) v1.6.1](https://www.nexusmods.com/monsterhunterworld/mods/2364)
- [Reshade Hook v1.0](https://www.nexusmods.com/monsterhunterworld/mods/7015)
- [Endemic Quality (Iceborne Edition) v1.4](https://www.nexusmods.com/monsterhunterworld/mods/2137)
- [Remove censor bad word banned word filter v1.0](https://www.nexusmods.com/monsterhunterworld/mods/6956)
  [Monster Weakness Icon Indicator for Iceborne (Hi-Res) vFinal](https://www.nexusmods.com/monsterhunterworld/mods/1938)
- [Cuter Handler Face Model (Post-Iceborne) aka Make Handler Cuter Again v3.0](https://www.nexusmods.com/monsterhunterworld/mods/1914)
</details>

**Comment by ioquirk** [comment]
Is the mod working: No, CTD on Filter Search/Matchmake.

**OS:** Microsoft Windows 11 Home 23H2 (OS Build 22631.3296)
**CPU:** AMD Ryzen 9 6900HS with Radeon Graphics | 3.30 GHz
**RAM:** 8 GM DDR5-4800 + 32 GB DDR5-4800
**GPU:** AMD Radeon RX 6800S 8GB GDDR6
**GPU Driver Version:** 23.40.27.01-240313a-401413C-AMD-Software-Adrenalin-Edition
**Game Version:** v15.21.00
**Stracker's Loader Version:** v3.0.1
**SharpPluginLoader Version:** v0.0.4.1

**Dxdiag Report File:** 
[DxDiag.txt](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14722408/DxDiag.txt)
**SharpPluginLoader Log File:** 
[SharpPluginLoader.log](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14722403/SharpPluginLoader.log)

**All Installed Mods (with versions):** 
- Stacker's Loader v3.0.1
- Performance Booster and Plugin Extender v1.22 Nov 2023 15.21 Patch
- Skippable Cutscenes v1.1
- Tic Rate Fix v1.4
- MHG Custom Quests FQ v1.4
- Lower Render Resolution (1080p) v1.0.1
- mhw_action_cloner_RELEASE.dll v1.0.2
- Guiding Lands Region Set v1.2
- True Weapon Damage Values v1.3.1
- Bow Sticky Aim Removal v1.2
- HunterPie V1 v1.0.6
- MHW ICE Experimental v314017 _(disabled in MHW folder)_

**Misc:** from Console popping up at game launch (with both Stracker's config set to "ERROR" and C Sharp Loader installed)
![Screenshot 2024-03-22 081045](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/assets/139350085/dbacf5b2-cb39-45ef-a862-c0fbc8761c08)

**Comment by iiOxygen** [comment]
**Is the mod working:** No
**If not then what happens:** CTD on Filter Search/Matchmake.

**OS:** Microsoft Windows 10 Home Version 22H2 (OS Build 19045.4170)
**CPU:** AMD Ryzen 9 3900X
**RAM:** 32GB 2133 MHz
**GPU:** AMD Radeon RX 5700 XT
**GPU Driver Version:** 31.0.21018.6011
**Game Version:** 15.21.00
**Stracker's Loader Version:** 3.0.1
**SharpPluginLoader Version:** 0.0.5

**Dxdiag Report File:** [DxDiag.txt](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14733885/DxDiag.txt)
**SharpPluginLoader Log File:** [SharpPluginLoader.log](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14733889/SharpPluginLoader.log)

**All Installed Mods (with versions):** 


<details>
<summary>Show/Hide</summary>

[Performance Booster and Plugin Extender 1.22](https://www.nexusmods.com/monsterhunterworld/mods/3473)

[Console Quick Operation 1.0.2](https://www.nexusmods.com/monsterhunterworld/mods/7043)

[Tic Rate Fix 1.4](https://www.nexusmods.com/monsterhunterworld/mods/3474)

[Fexty's Mod 1.5.4](https://www.nexusmods.com/monsterhunterworld/mods/4132)

[True weapon damage values 1.3.1](https://www.nexusmods.com/monsterhunterworld/mods/2459)

_some armour and weapon mods_
</details>

**Comment by GreenComfyTea** [comment]
I released a new test build, can you please try it: [https://github.com/GreenComfyTea/MHW-Better-Matchmaking/releases/tag/v2.0-amd-testing-v2](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/releases/tag/v2.0-amd-testing-v2)

**Comment by ioquirk** [comment]
:OO It is finally working without CTDs!

**Comment by GreenComfyTea** [comment]
Good to hear, but I would want to see a new report as above. I only fixed the crash but the mod is still shouldn't be working.

**Comment by ioquirk** [comment]
Alright. So far, the mod seems to be working completely fine. I'm able to see 32 results in Filtered Lobby Search and SOS's and see quest/lobbies from abroad (range set to Worldwide) with no crashes thus far. 

**Dxdiag Report File:** [DxDiag1.txt](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14736290/DxDiag1.txt)
**SharpPluginLoader Log File:** [SharpPluginLoader1.log](https://github.com/GreenComfyTea/MHW-Better-Matchmaking/files/14736291/SharpPluginLoader.log)

**Comment by GreenComfyTea** [comment]
Oh wait you are right. No errors in the log. I think I know what the issue is now, thanks.

**Comment by GreenComfyTea** [comment]
Potentially fixed in ca765942d6dd1f88ed020df3dd2282816b38d697. Released with https://github.com/GreenComfyTea/MHW-Better-Matchmaking/releases/tag/v2.1.

Will close later if I don't receive reports that the fix is not working. Until then if anyone wants to post their report here, please do so.
